hazelcast/src/test/java/com/hazelcast/cache/CacheConfigTest.java

```diff
@@ -203,6 +199,19 @@ public void defaultCacheTest() {
     }
 
     @Test
```

Comment:
:+1: I'm also on the way of learning the different styles - big thanks @pveentjer .